docs/general/server/media/media-segments.md

```diff
@@ -0,0 +1,46 @@
+---
+uid: server-media-media-segments
+title: Media Segments
+---
+
+# Media Segments
+
+Media segments are sections of media files with a defined type, stored within Jellyfin. They are different from chapters in that chapters don't have any defined type.
+
+## Types
+
+The following types are currently available:
+
+- Commercial
+- Preview
+- Recap
+- Outro
+- Intro
+
+## An example
+
+A video that is 16 minutes (00:16:00) long could have the following segments:
+
+1. From 00:00:00 until 00:03:08 there is an `Intro` segment.
+2. From 00:08:03 until 00:08:59 there is a `Commercial` segment.
+3. From 00:14:30 until 00:16:00 there is an `Outro` segment.
+
+Jellyfin can store this information and provide it via the MediaSegment API to clients. Clients can then decide what they wants to do with the provided information, such as displaying a "skip" button to allow convenient skipping.
+
+## Creating Media Segments
+
+As of Jellyfin 10.10, media segments can only be created by plugins. There is an official `Chapter Segments Provider` plugin that creates media segments from chapters.
```

Comment:
I think this is mostly covered below in the "Plugins" section, so perhaps unifying those two bits is better and this can be removed.

Reply:
moved the plugin mention to the plugins section